test1.2/task2/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "bogosort.h"
+
+int main()
+{
+    int len = 0;
+    int *array = NULL;
+    do
+    {
+        printf("Enter len. len > 0: ");
```

Comment:
```suggestion
        printf("Enter length (> 0): ");
```
Человекочитаемый вывод.